packages/language-support/src/signatureHelp.ts

```diff
@@ -59,13 +118,22 @@ function tryParseFunction(
     const methodName = ctx.functionName().getText();
     const numMethodArgs = ctx.functionArgument_list().length;
 
-    return {
+    result = {
       methodName: methodName,
       numProcedureArgs: numMethodArgs,
     };
-  } else {
-    return undefined;
+  } else if (currentNode.getText() === '(') {
```

Comment:
I think it'd be nice if we could move the comment from line 170 here instead to explain why we're looking for the `(` . Ideally it'd be nice to check that the current node is a child of statements here as well but it's fine to leave that check inside the helper fn if you prefer